Developer.md

```diff
@@ -0,0 +1,8 @@
+# SIMS Developer Notes
+
+Student Information Management System. Post-Secondary Student Financial Aid System.
+
+##IDP TEST USERS BCSC
+1. SS4BPS204 98904 FOUR
```

Comment:
I believe that we are not allowed to store this info here.
@abschwenker, can you please confirm it?

Reply:
Correct, please remove

Reply:
Done.